crypto/conf/conf_mod.c

```diff
@@ -11,6 +11,7 @@
 #include <ctype.h>
 #include <openssl/crypto.h>
 #include "internal/cryptlib.h"
+#include "e_os.h"
```

Comment:
Yes, there are two uses of OPENSSL_CONF in the code base.  Here and `apps/openssl.c`.
I was hoping to unify them rather than keeping them in a header.

If it is to move to a different header, which? `cryptlib.h`?